src/uwtools/rocoto.py

```diff
@@ -76,7 +76,7 @@ def _rocoto_template_xml() -> DefinitePath:
 
 def _write_rocoto_xml(
     config_file: OptionalPath,
-    rendered_output: DefinitePath,
+    rendered_output: OptionalPath,
```

Comment:
Hmm, while I get 'rendered' as a descriptor since we're creating it from a template, you have a fair point that its the path to the file that we're actually using. I think this would be a clearer change